torch/_torch_docs.py

```diff
@@ -970,6 +970,64 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.atanh`.
 """)
 
+add_docstr(torch.asarray,
+           r"""
+asarray(obj, *, dtype=None, device=None, copy=None, requires_grad=False) -> Tensor
+
+Creates a new :class:`Tensor` out of :attr:`obj` storage, by either copying or
+aliasing it.
+
+:attr:`obj` will be tested for, in order:
+
+1. a :class:`Tensor` object (copy and alias allowed)
```

Comment:
"a tensor"

Reply:
Can this convert NumPy arrays?

Reply:
Yes, but it handles NumPy arrays as python sequences. In other words, it always copies from NumPy arrays. Should we allow their storage to be shared (it should be straight-forward, since we already have `from_numpy`)?